Cognigy/chat-components
# Default Preview

This PR adds support for the _defaultPreview channel. Also updates dependencies reported by `npm audit`

To test:
1. `npm ci && npm pack`
2. deploy Webchat branch from this PR: https://github.com/Cognigy/Webchat/pull/30, 
3. install packed chat-components with `npm i ../chat-components/cognigy-chat-components-0.34.0.tgz`
4. `npm test`
5. `npm run dev`, check the default endpoint with different message types
